sycl/test/esimd/block_load_store.cpp

```diff
@@ -40,10 +40,19 @@ kernel3(accessor<int, 1, access::mode::write, access::target::device> &buf)
 
 // Incompatible mode (read).
 SYCL_EXTERNAL void
-kernel4(accessor<int, 1, access::mode::read, access::target::device> &buf)
+kernel5(accessor<int, 1, access::mode::read, access::target::device> &buf)
     SYCL_ESIMD_FUNCTION {
   simd<int, 32> v(0, 1);
   // CHECK: block_load_store.cpp:48{{.*}}error: no matching function
   // function for call to 'block_store'
   block_store<int, 32>(buf, 0, v);
 }
+
+// Incompatible mode (read).
+SYCL_EXTERNAL void
+kernel6(local_accessor<const int, 1> &buf) SYCL_ESIMD_FUNCTION {
```

Comment:
'const int' means it is read only. It is surprising that SYCL implementation allowed creating it because 
a) it is odd to have read-only local accessor - that memory cannot be written and thus is not usable 
b) SYCL 2020 SPEC says: "This specialization of accessor is only available for access modes access_mode::read_write and access_mode::atomic.".